docs/developer/extending/apps.mdx

```diff
@@ -1,8 +1,461 @@
+| MANAGE_CHECKOUTS            | Manage checkout                                                                                                                  |
+
+
+### Webhooks
+
+Webhooks can be used to receive data from Saleor when particular events happen. The available events are:
+
+| Name                        | Description                                                                                                                      |
+| --------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `ANY_EVENTS`                | Receive webhooks when any of the available events are triggered (wildcard event).                                                |
+| `CHECKOUT_QUANTITY_CHANGED` | Quantity of a checkout line is changed.                                                                                          |
+| `CUSTOMER_CREATED`          | A new customer account is created.                                                                                               |
+| `ORDER_CREATED`             | A new order is placed.                                                                                                           |
+| `ORDER_FULLY_PAID`          | Payment is made and an order is fully paid.                                                                                      |
+| `ORDER_UPDATED`             | An order is updated; triggered for all changes related to an order; covers all other order webhooks, except for `ORDER_CREATED`. |
+| `ORDER_CANCELLED`           | An order is cancelled.                                                                                                           |
+| `ORDER_FULFILLED`           | An order is fulfilled.                                                                                                           |
+| `FULFILLMENT_CREATED`       | A new fulfillment is created.                                                                                                    |
+| `PRODUCT_CREATED`           | A new product is created.                                                                                                        |
+
+Webhooks' payloads are sent in `POST` requests.
+
+### Difference between `create` and `install`
+- `create`: create a custom App instance, assign explicitly provided permissions. 
+The app is fully manageable. You can change assigned permissions, add webhooks, or auth_token. 
+`create` can be useful for creating integration with some services which will parse the webhook payload.
+- `install`: create App instance and prepare communication channel. 
+This approach uses a defined App manifest to prepare all required objects. 
+This approach is recommended for creating integration with a service dedicated to Saleor. 
```

Comment:
What is "a service dedicated to Saleor."? How is it different from the apps from the "create" example? TBH I don't understand the difference from reading this paragraph. From what I remember, in code we have two types of apps: local and third-party. Maybe we should explain the difference using these terms? I mean something like this:

## Types of apps in Saleor:
- local - definition, examples
- third-party - definition, examples

## Creating local apps
[...]

## Installing third-party apps
[...]

Probably an explanation of app types should be moved more to the top, maybe after "Architecture" part? It feels like it is mixed in the document right now.

Reply:
I think readers of this page will be people who either want to create and manage local apps or those who just want to install a third-party app. It should be easy for both groups to find the commands or API they need to use to do their job.